code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -99,9 +99,12 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
+		if(!iscultist(user) && !iswizard(user))
```

Comment:
if(!iscultist(user) && !iswizard(user) && !usability)

so that if you have a shard usable by anyone you can actually make constructs with it